packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -35,13 +35,14 @@ export class OdspDeltaStorageService {
      * @param from - inclusive
      * @param to - exclusive
      * @param telemetryProps - properties to add when issuing telemetry events
+     * @param reason - reason for fetching ops
      * @returns ops retrieved & info if result was partial (i.e. more is available)
      */
      public async get(
         from: number,
         to: number,
         telemetryProps: ITelemetryProperties,
-        fetchReason?: string,
+        reason?: string,
```

Comment:
In my latest change where we added this for some other calls, I used the name - "scenarioName". This tells the scenario in which this call is made. You may want to rename "reason" to that as reason does not tell you reason for what.

Reply:
I renamed args but kept `reason` in telemetry